### PR TITLE
Version 1.0

### PR DESCRIPTION
**Pull request by HugoSoszynski**
**CORE:**
- Fix `both` response mode is now only applied to the first filter in the chain 
- Fix when no next_filter is provided, set the filter parameter to `no` 
- Fix when no next filter is provided, no longer tries to connect to it 
- Debug mode now daemonize. Old debug mode is now named developer 
- Now cleans it's unix sockets 

**MANAGER:** 
- Fix swapped parameters in command builder 
- Fix missing clean after all the filter stop 
- Now waiting for the filter to daemonize before continuing 
- Fix missing default value in filter update 
- Added functions to check filter's files (pid, sock) 

**TOOLKIT:** 
- REDIS_MANAGER :: Add slave connection management 
- TIME :: Added GetTime() to toolkit folder 

**FSESSION:** 
- Refactor to use RedisManager instead of custom code 
- `evt-id` format in session improved 

**FTANOMALY:** 
- Fix, no longer add values empty values to redis 
- Improved log line format when incorrect input is given 

**FCONNECTION:** 
- Now parses input body 
- No longer cache 101 certitude 

**FLOGS:** 
- Smarter output stream management 

**TESTS:** 
- Manager's monitoring 
- Manager's updating 
- Manager and filter test toolkit available 

**MISC:** 
- Updated CMakeList.txt 
- Removed FreeBSD structure (compilation and specifics) from repository
- Removed FreeBSD package creation
- Greatly improved documentation
- Added PR and Issue templates 
- Added automated testing scripts 
- Removed access logs and replaced with debug logs 
- FILTERS :: Added 'filter' field to log line. Value is filter's name

### CODE REVIEW

File: samples/base/Session.cpp [hunk 1]
```diff
@@ -435,10 +446,10 @@ namespace darwin {
         return false;
     }
 
-    std::string Session::Evt_idToString(){
+    std::string Session::Evt_idToString() {
         char str[37] = {};
         sprintf(str,
```
Comment (KGuillemot) [inline samples/base/Session.cpp]:
```suggestion
        snprintf(str, 36, 
```

File: samples/flogs/LogsTask.cpp [hunk 2]
```diff
@@ -75,17 +77,26 @@ bool LogsTask::WriteLogs() {
...
+    if(fail) {
+        DARWIN_LOG_ERROR("LogsGenerator::LoadClassifier:: Error when opening the log file, "
+                         "too many retry, "
+                         "may due to low space disk or wrong permission");
         return false;
     }
 
-    logFile << body;
-    logFile.close();
+    _log_file << body << std::flush;
```
Comment (KGuillemot) [inline samples/flogs/LogsTask.cpp]: logFile.close() missing ?

Comment (HugoSoszynski) [reply samples/flogs/LogsTask.cpp]: No, it is part of the improvement.
Previously we closed the file after each input and opened it again to write the next one, now we keep the stream open until end or failure.

File: samples/base/Session.cpp (hunk 1 above)
Comment (HugoSoszynski) [reply samples/base/Session.cpp]: We need to test this before releasing it.

I will open an issue for the 1.0.1 milestone that concern the Core hardening.

**Pull request merged**